src/ElasticsearchMock/Client.php

```diff
@@ -411,16 +381,6 @@ public function reindex($params = [])
      *
      * @return array
      */
-    public function updateByQuery($params = [])
```

Comment:
@valentin-claras we should keep this I think
because a PR to make this method coming back was merged a few days ago : https://github.com/elastic/elasticsearch-php/pull/531/files#diff-ada47edcedf10150b8ea36ff1108d61fR1143